exec/java-exec/src/main/java/org/apache/drill/exec/store/ischema/InfoSchemaGroupScan.java

```diff
@@ -70,6 +67,12 @@ public InfoSchemaFilter getSchemaFilter() {
     return filter;
   }
 
+  @JsonIgnore
+  @Override
+  public List<SchemaPath> getColumns() {
+    return super.getColumns();
+  }
```

Comment:
I believe that we do need to serialize the columns as part of the logical plan. (Project push down occurs before creating the logical plan JSON.)

Reply:
Sys and InfoSchema plugins always produce all columns but added them to the plan to avoid confusion.

Reply:
Ah, I see. So, I'll amend my comment: we need to serialize columns only if the plugin supports project push-down. With EVF, that is all of them. But, of course, this one is old and may not support project push-down.

Reply:
Yes, these plugins do not support project push-down, I'll remove columns ser / de.